prime-dotnet-webapi/Controllers/EnrolleePaperSubmissionsController.cs

```diff
@@ -258,5 +268,72 @@ public async Task<ActionResult> FinalizeEnrolleePaperSubmission(int enrolleeId)
 
             return Ok();
         }
+
+        // HEAD: api/Enrollees/potential-paper-enrollee/5
+        /// <summary>
+        /// Gets all paper enrollees and checks whether or not there is a match in dateOfBirth with the current logged on enrollee.
+        /// </summary>
+        [HttpHead("potential-paper-enrollee/{dateOfBirth}")]
+        [Authorize(Roles = Roles.TriageEnrollee + "," + Roles.PrimeEnrollee)]
```

Comment:
do adjudicators need to hit this endpoint?

Reply:
They will at some point in the future have to link them manually. However, adjudicator actions are not part of this ticket though. I can remove the permission and later one we can add it when we work on the part where the adjudicator has to manually adjudicate and link the enrolments